tests/lookup/tests.py

```diff
@@ -204,6 +204,10 @@ def test_in_bulk(self):
         with self.assertRaises(TypeError):
             Article.objects.in_bulk(headline__startswith="Blah")
 
+    def test_in_bulk_values_iterable(self):
```

Comment:
I'm not in love with this method name, let me know if we want a tweak. `test_values_in_bulk`, `test_in_bulk_after_values` ,`test_values_before_in_bulk` all options

Reply:
Maybe?
```python
    def test_in_bulk_not_model_iterable(self):
        msg = "in_bulk() cannot be used with values() or values_list()."
        with self.assertRaisesMessage(TypeError, msg):
            Author.objects.values().in_bulk()
        with self.assertRaisesMessage(TypeError, msg):
            Author.objects.values_list().in_bulk()
```
to be put under `test_in_bulk_sliced_queryset`?